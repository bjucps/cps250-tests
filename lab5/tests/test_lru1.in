ape
bat
cat
ape
bat
cat
dog
bat
emu
dog
cat
