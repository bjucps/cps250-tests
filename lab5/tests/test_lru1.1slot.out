ape: MISS (1/1 full; no eviction)
bat: MISS (1/1 full; evicted 'ape')
cat: MISS (1/1 full; evicted 'bat')
ape: MISS (1/1 full; evicted 'cat')
bat: MISS (1/1 full; evicted 'ape')
cat: MISS (1/1 full; evicted 'bat')
dog: MISS (1/1 full; evicted 'cat')
bat: MISS (1/1 full; evicted 'dog')
emu: MISS (1/1 full; evicted 'bat')
dog: MISS (1/1 full; evicted 'emu')
cat: MISS (1/1 full; evicted 'dog')
--------------------------------
FINAL CACHE CONFIGURATION:
--------------------------------
cat
